client/src/client.rs

```diff
@@ -336,8 +336,26 @@ impl<DB: Database> Client<DB> {
         self.node.read().await.get_balance(address, block).await
     }
 
-    pub async fn get_nonce(&self, address: &Address, block: BlockTag) -> Result<u64> {
-        self.node.read().await.get_nonce(address, block).await
+    pub async fn get_transaction_count(&self, address: &Address, block: BlockTag) -> Result<u64> {
+        self.node
+            .read()
+            .await
+            .get_transaction_count(address, block)
+            .await
+    }
+
+    pub async fn get_block_transaction_count_by_hash(&self, hash: &Vec<u8>) -> Result<U256> {
```

Comment:
Can we have both of these functions just return `u64`. They are easier to work with, and it is impossible to have a block fill enough transactions to overflow this. Same goes for these functions in `Node`.